### PR TITLE
Update wording on milk jugs

### LINKED ISSUE #49563: Some sealed milk jugs spoil despite description suggesting they should not spoil
# Describe the bug

I found a sealed milk jug in a basement refrigerator which slowly went from fresh to old over a day. While it is expected that milk spoils over time, the description should make this more clear.

# Steps To Reproduce

Find a sealed jug of milk, and check if it spoils. Not all milk jugs seem to spoil though, some are "sealed" while others are "sealed fresh".

# Expected behavior

I expect all milk jugs to act the same, or at least for milk that does spoil/does not spoil to be labelled differently.

# Versions and configuration

 - OS: Windows 10
 - Game Version: bc28921 [64-bit]
 - Graphics version: Tiles
 - Ingame language: English
 - Mods loaded: [dda, no_npc_food, package_bionic_professions, Only_Wildlife]

# Additional context

If this is just a wording issue in the jug description, removal of "Some may be factory sealed to preserve content from rot until opened". If the different types of milk is intended, it should probably be "shelf-stable milk" instead.

[Spoiling Milk.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/6759461/Spoiling.Milk.zip)
[Stable Milk.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/6759462/Stable.Milk.zip)

### PR DESCRIPTION
**Pull request by ActualDio**
#### Summary
Bugfixes "Improve wording in plastic containers descriptions"

#### Purpose of change
Fixes #49563

#### Describe the solution
Simply replaced the wording on bottles and jugs' description

### CODE REVIEW

**Review by int-ua — APPROVED** [review]

**Comment by int-ua** [comment]
Please fix you Summary line. Should be 

```
Bugfixes "Update wording on milk jugs"
```

**Comment by ActualDio** [comment]
> 
> 
> Please fix your Summary line. Should be something like
> 
> ```
> Bugfixes "Improve wording in plastic containers descriptions"
> ```

Done, sorry about that

File: data/json/items/containers.json [hunk 1]
```diff
@@ -302,7 +302,7 @@
     "type": "GENERIC",
     "category": "container",
     "name": { "str": "plastic bottle" },
-    "description": "A watertight plastic bottle, holds 500 ml of liquid.  Some may be factory sealed to preserve content from rot until opened.",
+    "description": "A watertight plastic bottle, holds 500 ml of liquid.  Some may be factory sealed to increase shelf life.",
```
Comment (actual-nh) [inline data/json/items/containers.json:305]:
```suggestion
    "description": "A watertight plastic bottle, holds 500 ml of liquid.  Some may be factory-sealed to increase shelf life.",
```

File: data/json/items/containers.json [hunk 2]
```diff
@@ -1205,7 +1205,7 @@
     "category": "container",
     "name": { "str": "gallon jug" },
     "looks_like": "bottle_plastic",
-    "description": "A standard plastic jug used for milk and household cleaning chemicals.  Some may be factory sealed to preserve content from rot until opened.",
+    "description": "A standard plastic jug used for milk and household cleaning chemicals.  Some may be factory sealed to increase shelf life.",
```
Comment (actual-nh) [inline data/json/items/containers.json:1208]:
```suggestion
    "description": "A standard plastic jug used for milk and household cleaning chemicals.  Some may be factory-sealed to increase shelf life.",
```

**Comment by ActualDio** [comment]
Sorry for the mess I forgot to seperate my branches, and was bug fixing something else on my master branch. My bad

**Pull request merged**